packages/jest-types/src/Circus.ts

```diff
@@ -214,7 +214,7 @@ export type TestError = Exception | [Exception | undefined, Exception]; // the e
 export type TestEntry = {
   type: 'test';
   asyncError: Exception; // Used if the test failure contains no usable stack trace
-  errors: TestError;
+  errors: Array<TestError>;
```

Comment:
it was `any` before... `Array<any>` is more accurate, but still not particularly good 😅

Reply:
This is weird, now it can be an array of tuples? Look at `TestError`

Reply:
that fits reality
https://github.com/facebook/jest/blob/3375ac3b515e62ca1450e9a154ffdbdf7dc7e1f7/packages/jest-circus/src/eventHandler.ts#L166
https://github.com/facebook/jest/blob/3375ac3b515e62ca1450e9a154ffdbdf7dc7e1f7/packages/jest-circus/src/eventHandler.ts#L195

Reply:
you can see it used in `formatNodeAssertErrors.ts`